spec/lib/bookmark_manager_spec.rb

```diff
@@ -134,6 +136,17 @@ def create_bookmark
       expect(result[:topic_bookmarked]).to eq(true)
     end
 
+    context "if the bookmark is the last one bookmarked in the topic" do
+      before do
+        TopicUser.create(user: user, topic: bookmark.post.topic, bookmarked: true)
```

Comment:
why put this in a before block just for one test?

Reply:
...good point